cluster_itests/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '2'
+
+services:
+  zookeeper:
+    build: ../yelp_package/itest_dockerfiles/zookeeper/
+    ports:
+      - 2181
+
+  mesosmaster:
+    build: ../yelp_package/itest_dockerfiles/mesos/
+    image: paasta_itest_mesos:latest
+    ports:
+      - 5050
+    command: 'mesos-master --zk=zk://zookeeper:2181/mesos-testcluster --registry=in_memory --quorum=1 --authenticate --authenticate_slaves --work_dir=/tmp/mesos --credentials=/etc/mesos-secrets'
+    depends_on:
+      - zookeeper
+
+  mesosslave:
+    build: ../yelp_package/itest_dockerfiles/mesos/
+    ports:
+      - 5051
+    command: 'mesos-slave --master=zk://zookeeper:2181/mesos-testcluster --resources="cpus(*):10; mem(*):512; disk(*):100" --credential=/etc/mesos-slave-secret --containerizers=docker --docker=/usr/bin/docker --work_dir=/tmp/mesos --attributes="region:fakeregion;pool:default" --no-docker_kill_orphans --log_dir=/var/log/mesos'
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    depends_on:
+      - zookeeper
+
+  tronmaster:
+    build:
+      context: ../
+      dockerfile: ./yelp_package/itest_dockerfiles/tronmaster/Dockerfile
+    command: bash -c 'pip3 install -e /work && exec trond --debug -c /work/cluster_itests/tronfig/   -l /work/example-cluster/logging.conf'
```

Comment:
just to be consistent with example-cluster, can you rename config folder to `cluster_itests/config`?